core/background/pipeline/metadata.js

```diff
@@ -7,16 +7,30 @@ define([
 	return {
 		loadSong: function (song, cb) {
 
+			var isSongValid = false;
+
 			var onLoaded = function (isLfmValid) {
-				cb(isLfmValid);
+				isSongValid = isLfmValid;
 			};
 
-			let scrobbler = ScrobbleService.getFirstBound();
 
-			console.log('metadata loadSong() using: ' + scrobbler.getLabel());
+			let scrobblers = ScrobbleService.getAllBound();
+
+			for (var index in scrobblers) {
+				let scrobbler = scrobblers[index];
+				console.log('metadata loadSong() attempt using: ' + scrobbler.getLabel());
+
+				// load song metadata and update validation flag
+				scrobbler.loadSongInfo(song, onLoaded);
+
+				if (isSongValid) {
+					break;
```

Comment:
I wrote an example how it can be implemented: https://gist.github.com/alexesprit/f01e28ed41becc73e39bd217a7bc4df8
The callback will be called after first valid result of `loadSongInfo`.